projects/experimental/components/pdf-viewer/pdf-viewer.style.less

```diff
@@ -0,0 +1,80 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+tui-pdf-viewer {
+    display: block;
+    inline-size: 100%;
+    block-size: 100vh;
+    box-sizing: border-box;
+    color: var(--tui-text-primary);
+    padding-block-start: env(safe-area-inset-top);
+    padding-block-end: env(safe-area-inset-bottom);
+
+    --t-header: 4rem;
+
+    tui-root:not(._mobile) & {
+        background: var(--tui-background-accent-on-light, rgba(0, 0, 0, 1));
+    }
+
+    tui-root._mobile tui-dialog & {
+        --t-header: 2.5rem;
+
+        tui-app-bar[data-size] {
+            margin: calc(-1 * env(safe-area-inset-top)) 0 0;
+        }
+    }
+
+    tui-sheet-dialog & {
+        --t-header: 4.5rem;
+
+        tui-app-bar {
+            margin: -1rem -1rem 0;
+        }
+
+        .t-content {
+            margin: 0 -1rem;
+            inline-size: 100vw;
+        }
+    }
+
+    .t-header {
+        position: relative;
+        z-index: 1;
+        display: flex;
+        align-items: center;
+        block-size: 4rem;
+        padding: 0 0.5rem 0 1rem;
+    }
+
+    .t-title {
+        margin: 0;
+        font: var(--tui-font-text-m);
+        white-space: nowrap;
+        text-overflow: ellipsis;
+        overflow: hidden;
+    }
+
+    .t-actions {
```

Comment:
We can probably skip .t-actions altogether if we add gap: 0.5rem to .t-header and margin-inline-end: auto to .t-title